webhook/cert/manager_test.go

```diff
@@ -179,24 +179,22 @@ func TestMultipleManagers(t *testing.T) {
 }
 
 func TestOptionsSetDefault(t *testing.T) {
-	testcases := []struct {
+	testcases := map[string]struct {
```

Comment:
Let's not change the test style and remain consistent with the tests in the rest of the repo please.

Reply:
Does it mean i should replace the style everywhere, or revert it here?

Reply:
Just revert it from this test. No need to use a map. Have the name of the test case as a struct field.
Personally, I haven't seen the usage of this style in the new projects or k8s based projects, mostly. About 2 years ago, I remember seeing usage of map's key for setting the test names in table tests. But maybe it's just a coincidence that most of the projects I've worked on and read usually set explicit name field in the test struct. Just a styling preference :slightly_smiling_face:

Reply:
Ok, will revert it. But in my perspective this is not a styling question. Old or not, fancy or not it doesn't matter for me. My main let's say technical reason is data coupling.
- Map style is decoupling test data and test name while the other mixing those two different kind of data.
- Map style enforces to give a name to the test while the other doesn't.
- Map style visually separates cases so easier to seek in codebase.

Reply:
Reflecting back, I think the coupling of test name with data is usually done because the test name is used in the test assertion error message. I think before subtests were a thing, prepending the failure message with test name was helpful. But for this particular test, we aren't using the name. So, no need to change :smile: 
Thanks.

Reply:
The example in the subtest blog post also couples name with data and doesn't use it https://blog.golang.org/subtests :smile: 
But I think the reasoning is good. Keep them separate unless the name is needed in the test. I'll follow that from now on. Thanks a lot. :slightly_smiling_face: